src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -457,4 +457,19 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="AnalyzerTotalExecutionTime" xml:space="preserve">
+    <value>Total analyzer execution time: {0} seconds.</value>
+  </data>
+  <data name="MultithreadedAnalyzerExecutionNote" xml:space="preserve">
+    <value>NOTE: The actual wall clock time is likely lesser due to multithreaded analyzer execution.</value>
```

Comment:
I have reworded the text to: `NOTE: The actual wall clock time is likely lesser than the above execution time due to multithreaded analyzer execution.`

@JohnHamby let me know if you have a better suggestion.